src/ts/ContentMetadataBuilder.ts

```diff
@@ -76,7 +80,17 @@ export class ContentMetadataBuilder {
     this.contentMetadata.defaultResource = this.metadataOverrides.defaultResource || this.metadata.defaultResource;
     this.contentMetadata.streamUrl = this.metadataOverrides.streamUrl || this.metadata.streamUrl;
 
-    return this.contentMetadata;
+    const convivaContentInfo = {} as ConvivaMetadata;
+    convivaContentInfo[Conviva.Constants.ASSET_NAME] = this.contentMetadata.assetName;
+    convivaContentInfo[Conviva.Constants.ENCODED_FRAMERATE] = this.contentMetadata.encodedFrameRate;
+    convivaContentInfo[Conviva.Constants.DURATION] = this.contentMetadata.duration;
+    convivaContentInfo[Conviva.Constants.DEFAULT_RESOURCE] = this.contentMetadata.defaultResource;
+    convivaContentInfo[Conviva.Constants.STREAM_URL] = this.contentMetadata.streamUrl;
+    convivaContentInfo[Conviva.Constants.IS_LIVE] = this.contentMetadata.streamType;
```

Comment:
@gregrealeyes Since the name is IS_LIVE, should this be a boolean rather than the streamType string?

Reply:
In this new conviva library, Conviva.Constants.IS_LIVE = 'streamType' so I think we have to leave this as is. Conviva.Constants.StreamType are 3 string options that need to be set to islive or the touchstone session throws an integration error. It is super confusing
Conviva.Constants.IS_LIVE
"Conviva.streamType"

Reply:
weird. ok